code/datums/progressbar.dm

```diff
@@ -10,11 +10,12 @@
 
 /datum/progressbar/New(mob/User, goal_number, atom/target)
 	. = ..()
-	if(!istype(target))
+	if (!istype(target))
```

Comment:
:eyes:

Please replace all of the spaces between `if` and the opening brackets `(`. This is why copypasting whole files is bad :eyes: